src/content/news/2025-08-04-maplibre-newsletter-july-2025/index.mdx

```diff
@@ -0,0 +1,346 @@
+---
+title: "MapLibre Newsletter July 2025"
+date: "2025-08-04"
+categories: ["newsletter"]
+authors: [bart, harel, yuri, frank, ramyaragupathy]
+draft: false
+---
+
+import { Image } from "astro:assets";
+import jawgmapsLogo from "/public/img/jawgmaps-logo.svg";
+import maplibreLogo from "/public/img/maplibre-logos/maplibre-logo-for-light-bg.svg";
+import myanmarImg from "../../../content/roadmap/writing-systems/myanmar.png";
+import thaiImg from "../../../content/roadmap/writing-systems/thai.png";
+import martinImg from "./martin-ui.png";
+import nativeRsSocialImg from "./maplibre-native-rs.png";
+import foss4gImg from "./foss4g-nl.png";
+import sotmusImg from "./sotm-us.png";
+
+Welcome to the July edition of the MapLibre newsletter! We start with a special thanks to our long-term sponsor Jawg Maps (2023–2025) 🙏 for their continued support and commitment to open mapping.
+
+<div
+  style={{
+    backgroundColor: "white",
+    padding: "2rem",
+    border: "10px solid #cfd8dc",
+    borderRadius: "0",
+    marginBottom: "3rem",
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+    gap: "2rem",
+    flexWrap: "nowrap",
+    overflow: "hidden",
+    flexDirection: "column",
+  }}
+>
+  <div
+    style={{
+      display: "flex",
+      justifyContent: "center",
+      alignItems: "center",
+      gap: "2rem",
+      flexWrap: "nowrap",
+      width: "100%",
+    }}
+  >
+    <Image
+      src={maplibreLogo}
+      alt="MapLibre logo"
+      style={{
+        width: "40%",
+        maxWidth: "300px",
+        height: "auto",
+        objectFit: "contain",
+      }}
+    />
+    <Image
+      src={jawgmapsLogo}
+      alt="Mierune logo"
+      style={{
+        width: "40%",
+        maxWidth: "330px",
+        height: "auto",
+        objectFit: "contain",
+      }}
+    />
+  </div>
+  <div
+    style={{
+      marginTop: "1rem",
+      fontSize: "1rem",
+      textAlign: "center",
+      color: "#263238",
+      fontStyle: "italic",
+    }}
+  >
+    Sincere thanks to Jawg Maps for their ongoing commitment to MapLibre
```

Comment:
This is repeated a bit before and after the image, might be too soon...?

Reply:
wait - modifying a bit, removing the intro line and retaining it in the image.